schemas/hash.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: hash
+  title: Group
+  group: 2
+  type: group
+  short: Hashes, usually file hashes.
+  description: >
+    The hash fields represent different hash algorithms and their values.
```

Comment:
I recommend establishing an expected format of the hashes such as lowercase hex.